jrnl/cli.py

```diff
@@ -140,7 +140,7 @@ def run(manual_args=None):
     configure_logger(args.debug)
     args.text = [p.decode('utf-8') if util.PY2 and not isinstance(p, unicode) else p for p in args.text]
     if args.version:
-        version_str = "{0} version {1}".format(jrnl.__title__, jrnl.__version__)
+        version_str = "{0} version {1} (Python {3})".format(jrnl.__title__, jrnl.__version__, sys.version)
```

Comment:
This throws an error currently: 

```suggestion
        version_str = "{0} version {1} (Python {2})".format(jrnl.__title__, jrnl.__version__, sys.version)
```